### DISCUSSION TITLE
Is this a suitable ESP32 ?

### DISCUSSION THREAD

**Discussion by B-Hartley** [ESPresense/ESPresense#162] [Q&A]
Hi,

I'm new to ESPresence and new to ESP32.
Is this suitable for the project?

[SP-Cow ESP32S ESP32 Development Board 2.4GHz Dual-Mode WiFi+ Bluetooth Dual Cores Microcontroller Processor, Antenna Module with RF AMP Filter AP STA IDE Ultra Low Power](https://www.amazon.co.uk/SP-Cow-Development-Dual-Mode-Bluetooth-Microcontroller/dp/B07VJ34N2Q/ref=sr_1_16_sspa)

Thanks.

**Comment by SPYUK** [comment]
yes

**Reply by dbeckers99** [reply]
I have exactly this device - but the firmware install is not possible. There is a hang after starting install.

How can I install the firmware from shell (without WEB UI) and watch/debug what is going wrong?

**Reply by AlfredJKwack** [reply]
I had that happen. Double check you've installed the correct drivers for that specific device. 

Releases are located [here](https://github.com/ESPresense/ESPresense/releases). You could probably figure out how to install it manually from docs located at [ESPhome](https://esphome.io/guides/getting_started_command_line.html). However since you asked specifically how to get to get a terminal interface going, that's already documented [in the troubleshooting docs](https://espresense.com/troubleshooting/logs#esp32-via-serial).

**Reply by dbeckers99** [reply]
I figured out the problem: obviously a mismatching driver or configuration on my MacBook. On my MacStudio it works fine. Strange...

**Reply by AlfredJKwack** [reply]
Yep. That happens.

**Comment by gnome4711** [comment]
Those will do as well!!

[https://www.amazon.de/AZDelivery-Bluetooth-Internet-Entwicklungsboard-kompatibel/dp/B08BTLYSTM/ref=sr_1_1_sspa?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=1ERZ68XW0DHP6&keywords=AZDelivery+3+x+ESP32+D1+Mini&qid=1663528599&s=industrial&sprefix=azdelivery+3+x+esp32+d1+mini+%2Cindustrial%2C122&sr=1-1-spons&psc=1&smid=A1X7QLRQH87QA3](url)

I#m using them in my house. They work great!

They are smaller (if size matters  ;-)  )

**Comment by AlfredJKwack** [comment]
This one also works quite nicely: D1 mini esp32 CH9102X 
https://www.aliexpress.com/item/32821603622.html
